app/models/form_submission_attempt.rb

```diff
@@ -25,14 +25,21 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        log_info = { form_submission_id:,
+                     benefits_intake_uuid: form_submission&.benefits_intake_uuid,
+                     form_type: form_submission&.form_type }
         if should_send_simple_forms_email
-          Rails.logger.info({
-                              message: 'Preparing to send Form Submission Attempt error email',
-                              form_submission_id:,
-                              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
-                            })
+          Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
           simple_forms_enqueue_result_email(:error)
+        else
```

Comment:
Do we want to change this to something like `elsif should_send_form526_form4142_email` instead of a catch-all `else`?

Reply:
oh the commenting messed this up, this is supposed to be on the previous line and yes this should be an elsif